common/types.ts

```diff
@@ -24,3 +24,347 @@ export interface LibraryAddresses {
 export interface LinkReferences {
   [libraryLabel: string]: Array<{ start: number, length: number }>;
 }
+
+export interface SolJson {
```

Comment:
Hey @stephensli, we decided that this can be considered out of the scope of this PR. We can make this change later. So please ignore this comment here for now. I hope you didn't spent time on it :)

Let me know if you need any help to finish this PR so we can get it merged :D

Reply:
Sounds good, i'll try and find some time during the week to complete the remainder otherwise it will be completed on the weekend.